packages/core/__tests__/warnings.js

```diff
@@ -206,3 +206,39 @@ test('unterminated comments', () => {
     `"Your styles have an unterminated comment (\\"/*\\" without corresponding \\"*/\\")."`
   )
 })
+
+test('keyframes interpolated into plain string', () => {
+  const animateColor = keyframes({
+    'from,to': { color: 'green' },
+    '50%': { color: 'hotpink' }
+  })
+  const rotate360 = keyframes({
+    from: {
+      transform: 'rotate(0deg)'
+    },
+    to: {
+      transform: 'rotate(360deg)'
+    }
+  })
+
+  renderer.create(
+    <div css={[`animation: ${animateColor} 10s ${rotate360} 5s;`]} />
+  )
+  expect(console.error.mock.calls).toMatchInlineSnapshot(`
+    Array [
+      Array [
+        "\`keyframes\` output got interpolated into plain string, please wrap it with \`css\`.
+
+    Instead of doing this:
+
+    const animation0 = {from,to{color:green;}50%{color:hotpink;}}
+    const animation1 = {from{transform:rotate(0deg);}to{transform:rotate(360deg);}}
```

Comment:
The keyframes should probably be wrapped in a keyframe call.

Reply:
right, right, had that in mind, forgot to put it into the code 😅 fixed!